### PR TITLE
New package: python-iptools-0.6.1

### PR DESCRIPTION
**Pull request by genericly**

### CODE REVIEW

File: srcpkgs/python-iptools/template [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	}
+}
+
+python3.4-iptools_package() {
+	noarch=yes
+	build_style=meta
+	short_desc="${short_desc/Python2/Python3.4} (transitional dummy package)"
+	depends="python3-iptools>=${version}_${revision}"
+	vlicense LICENSE
+}
```
Comment (Vaelatern) [inline srcpkgs/python-iptools/template]: Please remove the `3.4` package and symlink. We no longer use them.

File: srcpkgs/python-iptools/template [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+depends="python"
+short_desc="Python utilites for manipulating IPv4 and IPv6 addresses"
+maintainer="Andrew Mirghassemi <a.mirghassemi@live.com>"
+license="2-clause-BSD"
+homepage="https://github.com/bd808/python-iptools"
+distfiles="${PYPI_SITE}/i/iptools/iptools-${version}.tar.gz"
+checksum=0f03875a5bed740ba4bf44decb6a78679cca914a1ee8a6cc468114485c4d98e3
+
+post_install() {
+
```
Comment (Vaelatern) [inline srcpkgs/python-iptools/template]: Remove spare newline please

File: srcpkgs/python-iptools/template [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+pkgname=python-iptools
+version=0.6.1
+revision=1
+noarch=yes
+wrksrc="${pkgname#*-}-${version}"
+build_style=python-module
+pycompile_module="iptools"
+hostmakedepends="python-setuptools python3-setuptools"
+depends="python"
+short_desc="Python utilites for manipulating IPv4 and IPv6 addresses"
```
Comment (Vaelatern) [inline srcpkgs/python-iptools/template]: `Python2 utilities for manipulating IPv4 and IPv6 addresses`

**Comment by genericly** [comment]
Fixed

**Comment by the-maldridge** [comment]
:shipit:

**Pull request closed without merging**